assets/javascripts/discourse/models/custom-wizard.js.es6

```diff
@@ -8,7 +8,7 @@ import { popupAjaxError } from 'discourse/lib/ajax-error';
 
 const CustomWizard = EmberObject.extend({
   save(opts) {
-    return new Promise((resolve, reject) => {      
+    return new Promise((resolve, reject) => {
```

Comment:
@fzngagan Changes to a file that are just removal of whitespace should not be in the PR

Reply:
Ok, I've reverted this change from the PR branch. :)